packages/docs/PropTables/PillTabsPropTable.tsx

```diff
@@ -11,12 +11,12 @@ const pillTabsPropTable: Prop[] = [
   {
     description: (
       <>
-        See <NextLink href="#pill-tabs-items-prop-table">below</NextLink> for usage.
+        See <NextLink href="/pill-tabs?props=pill-tab-item#props">here</NextLink> for usage.
```

Comment:
We did something a little differently for the other pages. See the [TablePropTable](https://github.com/bigcommerce/big-design/blob/master/packages/docs/PropTables/TablePropTable.tsx#L11) as an example.

Reply:
sounds good, I'll take a look. Ty!